playground/components/header.tsx

```diff
@@ -61,88 +71,109 @@ export const Header: Component<{
           <img src={logo} alt="solid-js logo" class="w-8" />
         </a>
         <span class="inline-block -mb-1">Solid Playground</span>
-      </h1>
-
+      </h1> 
       <div class="flex items-center space-x-2">
-        <button
-          type="button"
-          onClick={props.toggleDark}
-          class="px-3 py-2 focus:outline-none focus:ring-1 rounded text-white opacity-80 hover:opacity-100"
-          title="Toggle dark mode"
-        >
-          <span class="sr-only">{props.dark ? 'Light' : 'Dark'} mode</span>
-
-          <Show
-            when={props.dark}
-            fallback={
-              <svg class="h-6" fill="none" viewBox="0 0 24 24" stroke="currentColor">
-                <path
-                  stroke-linecap="round"
-                  stroke-linejoin="round"
-                  stroke-width="2"
-                  d="M20.354 15.354A9 9 0 018.646 3.646 9.003 9.003 0 0012 21a9.003 9.003 0 008.354-5.646z"
-                />
+        <div
+          class={`z-10`}
+          style={(showMenu()) ? {
+            position: "absolute",
+            top: "56px", //Set based on header size+margin*2
+            width: "fit-content"
+          } : {}}>
```

Comment:
This can be written as follow:
```suggestion
         classList={{ 'absolute top-[56px] w-[fit-content]':  showMenu() }}
```